src/base/bittorrent/session.h

```diff
@@ -603,6 +607,7 @@ namespace BitTorrent
 #if defined(Q_OS_WIN)
         void applyOSMemoryPriority() const;
 #endif
+        void processTrackerStatuses();
```

Comment:
Is it reasonable to declare `m_updatedTrackerEntries` as `mutable` and make this function `const`?
It seems to me that `m_updatedTrackerEntries` is just an temporary buffer.